mobile/android/build.gradle

```diff
@@ -1,5 +1,6 @@
 ext {
-    appCompatVersion = '1.1.0' // for background_fetch
+    compileSdkVersion   = 35
+    targetSdkVersion    = 35
```

Comment:
Can you confirm if targetSDK change really required?
From the background_fetch project docs, it looks like we don't need to add appCompactVersion pinning anymore.

But changing the targetSDK will require us to test the app for any change in behaviour.

https://developer.android.com/about/versions/15/behavior-changes-15

What was the previous targetSDK version which was coming via flutter?